audeer/core/utils.py

```diff
@@ -573,6 +574,52 @@ def _worker():
     return results
 
 
+def sort_versions(
+        versions: List[str],
+) -> List:
+    """Sort version numbers.
+
+    If the version starts with ``'v'``
+    it is ignored during sorting.
+
+    Args:
+        versions: sequence with semantic version numbers
+
+    Returns:
+        sorted list of versions with highest as last entry
+
+    Raises:
+        ValueError: if the version does not comply
+            with :func:`is_semantic_version`
+
+    Example:
+        >>> vers = [
+        ...     '2.0.0',
+        ...     '2.0.1',
+        ...     'v1.0.0',
+        ...     'v2.0.0-1-gdf29c4a',
+        ... ]
+        >>> sort_versions(vers)
+        ['v1.0.0', '2.0.0', 'v2.0.0-1-gdf29c4a', '2.0.1']
+
+    """
+    for version in versions:
+        if not is_semantic_version(version):
+            raise ValueError(
+                "All version numbers have to semantic versions, "
+                "following 'X.Y.Z', "
+                "where X, Y, Z are integers. "
+                f"Your vesion is instead: '{version}'."
```

Comment:
```suggestion
                f"Your version is: '{version}'."
```

Reply:
I updated this manually as I also test for the error message and had to update the text there as well.

Reply:
Thanks!